tests/testthat/test-npn-observations.R

```diff
@@ -122,6 +136,20 @@ test_that("phenometrics downloads work", {
 
   expect_gt(num_mag_custom, num_mag_default)
 
+  vcr::use_cassette("npn_download_no_records", {
```

Comment:
This sets up "mocking" so the next time the test is run it won't actually query the API, but instead use a response stored in the tests/fixtrues folder.